runbot_send_email/models/runbot_build.py

```diff
@@ -0,0 +1,167 @@
+# © 2016 Vauxoo
+#   Coded by: lescobar@vauxoo.com
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import logging
+import re
+from urllib.parse import urlparse
+from email.utils import formataddr
+from odoo import _, api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class RunbotBuild(models.Model):
+    # pylint: disable=method-compute
+    _name = "runbot.build"
+    _inherit = ['runbot.build', 'mail.thread']
+
+    repo_link = fields.Char()
```

Comment:
Delete all these fields because this information could be extracted from runbot.build frontend view by the user.